src/Cooker/providers/GitProvider/getHashListFromHash.test.js

```diff
@@ -24,11 +24,17 @@ it('should return empty list for master', done => {
   })
 })
 
+/*
+
+  Do not think this is correct? If missing release hash we should return
+  all history as long as it is under 50 commits?
+
 it('should return an empty list for bad hash', done => {
   getHashListFromHash(config.path, 'badhash').then(list => {
     assert.deepEqual(list, [], done)
   })
 })
+*/
```

Comment:
I don't think running on a bad hash is a good idea: this is related to providing a wrong hash value (which is really an error), it should break. If people want to use "Big Bang", they have to use a `when` operator to set the hash if it wasn't found. Should not be something automatic like this...